### ISSUE TITLE
Key input/typeahead functionality

### ISSUE THREAD

**Issue by ttacon** [b1lly/gumi#34]
Is this something that could be considered for gumi? Or is it advised to use a different plugin for this type of behaviour?

**Comment by b1lly** [comment]
Yes, keyboard interactions is definitely something I intended to support.

**Comment by ttacon** [comment]
Any timeline or are PRs welcome?

**Comment by b1lly** [comment]
There's no timeline. PR are always welcome :+1:
